<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cell_personInfoClass</class>
 <widget class="QWidget" name="cell_personInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cell_personInfo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
border:2px solid gray;
border-radius:10px;
padding:5px;
background: white;
selection-background-color: darkgray;
width:150px;
height:15px;
}
QLabel {
    font-family: &quot;Microsoft YaHei&quot;;

}</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>      读者个人信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>474</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>读者学号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numLE"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>读者姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLE"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>读者专业：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="majorLE"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>读者班级：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="classLE"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>联系方式：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phoneLE"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="BookMS.qrc">:/Character/spiderman.png</pixmap>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BookMS.qrc"/>
 </resources>
 <connections/>
</ui>
